qiskit/schemas/qobj_schema.json

```diff
@@ -847,6 +847,41 @@
             },
             "type": "array"
         },
+        "calibration_entry": {
+            "description": "The instruction description of each calibrated gate.",
+            "type": "object",
+            "properties": {
+                "name": {
+                    "description": "Gate name",
+                    "type": "string"
+                },
+                "qubits": {
+                    "items": {
+                        "minimum": 0,
+                        "type": "integer"
+                    },
+                    "minItems": 1,
+                    "type": "array"
+                },
+                "params": {
+                    "description": "Gate parameters",
+                    "type": "array"
```

Comment:
What are the types of these entries? I believe we should add support for parameters now, it is always much harder to migrate a schema then it is normal source code and the enhancements for parameters would be relatively minimal (the addition of a typed `args` section and making `params` optional. Instructions could then be extended in the future in a non-breaking manner to accept an arg specified in `args`.

Of course given the above it should be possible to extend calibration entries to support arguments by makings `params` -optional in the future and adding `args`. So feel free to ignore this comment.

Reply:
@ajavadia and I talked about this today. This does now support parameters as they are already used in circuits (bound parameters). Aer backends seem to support unbound parameters; Ali is going to start a conversation with Chris Wood about this. We can't yet send freely parameterized pulse qobj's, and this calibration is basically the heart of a mini pulse qobj. When parameters can be sent with a qobj, it should work whether it's a gate param, a pulse gate param, or a schedule param. Something of that scope should have a design doc to get input from the team.

As you said, it's possible to add free parameters as a fully backwards compatible feature. Given that, I'd like to focus this on the feature and optimize qobj size in a separate work.

Ali do you have a comment? Does Thomas' comment change your opinion?